configure uses AC_CHECK_FILE to test for file existence which is
cross incompatible. Changed it to simple if/else statements.

--- configure.orig	2017-10-26 19:44:39.000000000 +0200
+++ configure	2017-11-09 21:23:45.099433701 +0100
@@ -20200,29 +20200,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/expat/lib/expat.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/expat/lib/expat.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/expat/lib/expat.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/expat/lib/expat.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  EXPAT_LOCAL_AVAILABLE="yes"
-else
-  EXPAT_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/expat/lib/expat.h"; then
+                 EXPAT_LOCAL_AVAILABLE="yes"
+   else
+                 EXPAT_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$EXPAT_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: Expat libraries are available in the local tree" >&5
@@ -20455,53 +20437,17 @@
 $as_echo "$as_me: FFmpeg library NOT available as system library" >&6;}
    fi
 
-         as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/ffmpeg/libavcodec/avcodec.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/ffmpeg/libavcodec/avcodec.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/ffmpeg/libavcodec/avcodec.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/ffmpeg/libavcodec/avcodec.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  avcodec_h_found="yes"
-else
-  avcodec_h_found="no"
-fi
-
-
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/ffmpeg/libavformat/avformat.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/ffmpeg/libavformat/avformat.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/ffmpeg/libavformat/avformat.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/ffmpeg/libavformat/avformat.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  avformat_h_found="yes"
-else
-  avformat_h_found="no"
-fi
+         if test -f "${srcdir}/lib-src/ffmpeg/libavcodec/avcodec.h"; then
+                 avcodec_h_found="yes"
+   else
+                 avcodec_h_found="no"
+   fi
 
+   if test -f "${srcdir}/lib-src/ffmpeg/libavformat/avformat.h"; then
+                 avformat_h_found="yes"
+   else
+                 avformat_h_found="no"
+   fi
 
    if test "$avcodec_h_found" = "yes" -a "$avformat_h_found" = "yes"; then
       FFMPEG_LOCAL_AVAILABLE="yes"
@@ -20589,29 +20535,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/lame/lame/lame.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/lame/lame/lame.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/lame/lame/lame.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/lame/lame/lame.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LAME_LOCAL_AVAILABLE="yes"
-else
-  LAME_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/lame/lame/lame.h"; then
+                 LAME_LOCAL_AVAILABLE="yes"
+   else
+                 LAME_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LAME_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: LAME headers are available in this source tree." >&5
@@ -20780,53 +20708,17 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libflac/include/FLAC/format.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libflac/include/FLAC/format.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libflac/include/FLAC/format.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libflac/include/FLAC/format.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  flac_h_available="yes"
-else
-  flac_h_available="no"
-fi
-
-
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libflac/include/FLAC++/decoder.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libflac/include/FLAC++/decoder.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libflac/include/FLAC++/decoder.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libflac/include/FLAC++/decoder.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  flacpp_h_available="yes"
-else
-  flacpp_h_available="no"
-fi
+   if test -f "${srcdir}/lib-src/libflac/include/FLAC/format.h"; then
+                 flac_h_available="yes"
+   else
+                 flac_h_available="no"
+   fi
 
+   if test -f "${srcdir}/lib-src/libflac/include/FLAC++/decoder.h"; then
+                 flacpp_h_available="yes"
+   else
+                 flacpp_h_available="no"
+   fi
 
    if test "$flac_h_available" = "yes" -a "$flacpp_h_available" = "yes"; then
       LIBFLAC_LOCAL_AVAILABLE="yes"
@@ -20930,29 +20822,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libid3tag/frame.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libid3tag/frame.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libid3tag/frame.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libid3tag/frame.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBID3TAG_LOCAL_AVAILABLE="yes"
-else
-  LIBID3TAG_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/libid3tag/frame.h"; then
+                 LIBID3TAG_LOCAL_AVAILABLE="yes"
+   else
+                 LIBID3TAG_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBID3TAG_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: libid3tag libraries are available in the local tree" >&5
@@ -21099,29 +20973,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libmad/frame.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libmad/frame.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libmad/frame.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libmad/frame.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBMAD_LOCAL_AVAILABLE="yes"
-else
-  LIBMAD_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/libmad/frame.h"; then
+                 LIBMAD_LOCAL_AVAILABLE="yes"
+   else
+                 LIBMAD_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBMAD_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: libmad libraries are available in the local tree" >&5
@@ -21144,29 +21000,11 @@
          LIBNYQUIST_SYSTEM_AVAILABLE="no"
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libnyquist/nyx.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libnyquist/nyx.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libnyquist/nyx.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libnyquist/nyx.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBNYQUIST_LOCAL_AVAILABLE="yes"
-else
-  LIBNYQUIST_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/libnyquist/nyx.h"; then
+                 LIBNYQUIST_LOCAL_AVAILABLE="yes"
+   else
+                 LIBNYQUIST_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBNYQUIST_LOCAL_AVAILABLE" = "yes" ; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: nyquist libraries are available in the local tree" >&5
@@ -21265,29 +21103,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/sbsms/include/sbsms.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/sbsms/include/sbsms.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/sbsms/include/sbsms.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/sbsms/include/sbsms.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBSBSMS_LOCAL_AVAILABLE="yes"
-else
-  LIBSBSMS_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/sbsms/include/sbsms.h"; then
+                 LIBSBSMS_LOCAL_AVAILABLE="yes"
+   else
+                 LIBSBSMS_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBSBSMS_LOCAL_AVAILABLE" = "yes"; then
             LIBSBSMS_LOCAL_CONFIGURE_ARGS="--disable-programs"
@@ -21388,29 +21208,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libsndfile/src/sndfile.h.in" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libsndfile/src/sndfile.h.in" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libsndfile/src/sndfile.h.in... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libsndfile/src/sndfile.h.in"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBSNDFILE_LOCAL_AVAILABLE="yes"
-else
-  LIBSNDFILE_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/libsndfile/src/sndfile.h.in"; then
+                 LIBSNDFILE_LOCAL_AVAILABLE="yes"
+   else
+                 LIBSNDFILE_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBSNDFILE_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: libsndfile libraries are available in this source tree" >&5
@@ -21650,29 +21452,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/soundtouch/include/SoundTouch.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/soundtouch/include/SoundTouch.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/soundtouch/include/SoundTouch.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/soundtouch/include/SoundTouch.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBSOUNDTOUCH_LOCAL_AVAILABLE="yes"
-else
-  LIBSOUNDTOUCH_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/soundtouch/include/SoundTouch.h"; then
+                 LIBSOUNDTOUCH_LOCAL_AVAILABLE="yes"
+   else
+                 LIBSOUNDTOUCH_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBSOUNDTOUCH_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: libsoundtouch libraries are available in the local tree" >&5
@@ -21771,29 +21555,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libsoxr/src/soxr.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libsoxr/src/soxr.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libsoxr/src/soxr.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libsoxr/src/soxr.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBSOXR_LOCAL_AVAILABLE="yes"
-else
-  LIBSOXR_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/libsoxr/src/soxr.h"; then
+                 LIBSOXR_LOCAL_AVAILABLE="yes"
+   else
+                 LIBSOXR_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBSOXR_LOCAL_AVAILABLE" = "yes"; then
       # Breaks other other libraries in Audacity tree; but why is ./configure
@@ -21894,29 +21660,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/twolame/libtwolame/twolame.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/twolame/libtwolame/twolame.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/twolame/libtwolame/twolame.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/twolame/libtwolame/twolame.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBTWOLAME_LOCAL_AVAILABLE="yes"
-else
-  LIBTWOLAME_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/twolame/libtwolame/twolame.h"; then
+                 LIBTWOLAME_LOCAL_AVAILABLE="yes"
+   else
+                 LIBTWOLAME_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBTWOLAME_LOCAL_AVAILABLE" = "yes"; then
             LIBTWOLAME_LOCAL_CONFIGURE_ARGS="--disable-programs"
@@ -22015,29 +21763,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libvamp/vamp-hostsdk/PluginLoader.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libvamp/vamp-hostsdk/PluginLoader.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libvamp/vamp-hostsdk/PluginLoader.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libvamp/vamp-hostsdk/PluginLoader.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LIBVAMP_LOCAL_AVAILABLE="yes"
-else
-  LIBVAMP_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/libvamp/vamp-hostsdk/PluginLoader.h"; then
+                 LIBVAMP_LOCAL_AVAILABLE="yes"
+   else
+                 LIBVAMP_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LIBVAMP_LOCAL_AVAILABLE" = "yes"; then
       LIBVAMP_LOCAL_CONFIGURE_ARGS="--disable-programs"
@@ -22136,52 +21866,17 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libvorbis/include/vorbis/vorbisenc.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libvorbis/include/vorbis/vorbisenc.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libvorbis/include/vorbis/vorbisenc.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libvorbis/include/vorbis/vorbisenc.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  vorbisenc_h_available="yes"
-else
-  vorbisenc_h_available="no"
-fi
-
+   if test -f "${srcdir}/lib-src/libvorbis/include/vorbis/vorbisenc.h"; then
+                 vorbisenc_h_available="yes"
+   else
+                 vorbisenc_h_available="no"
+   fi
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/libogg/include/ogg/ogg.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/libogg/include/ogg/ogg.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/libogg/include/ogg/ogg.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/libogg/include/ogg/ogg.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  ogg_h_available="yes"
-else
-  ogg_h_available="no"
-fi
+   if test -f "${srcdir}/lib-src/libogg/include/ogg/ogg.h"; then
+                 ogg_h_available="yes"
+   else
+                 ogg_h_available="no"
+   fi
 
 
    if test "$vorbisenc_h_available" = "yes" -a "$ogg_h_available" = "yes"; then
@@ -22290,29 +21985,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/lv2/configure" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/lv2/configure" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/lv2/configure... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/lv2/configure"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  LV2_LOCAL_AVAILABLE="yes"
-else
-  LV2_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/lv2/configure"; then
+                 LV2_LOCAL_AVAILABLE="yes"
+   else
+                 LV2_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$LV2_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: LV2 libraries are available in the local tree" >&5
@@ -22432,29 +22109,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/portaudio-v19/include/portaudio.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/portaudio-v19/include/portaudio.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/portaudio-v19/include/portaudio.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/portaudio-v19/include/portaudio.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  PORTAUDIO_LOCAL_AVAILABLE="yes"
-else
-  PORTAUDIO_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/portaudio-v19/include/portaudio.h"; then
+                 PORTAUDIO_LOCAL_AVAILABLE="yes"
+   else
+                 PORTAUDIO_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$PORTAUDIO_LOCAL_AVAILABLE" = "yes"; then
                   pa_dir="$(pwd)/${srcdir}/lib-src/portaudio-v19"
@@ -22555,29 +22214,12 @@
 $as_echo "$as_me: portSMF library is NOT available as system library" >&6;}
    fi
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/portsmf/allegro.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/portsmf/allegro.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/portsmf/allegro.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/portsmf/allegro.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  PORTSMF_LOCAL_AVAILABLE="yes"
-else
-  PORTSMF_LOCAL_AVAILABLE="no"
-fi
 
+   if test -f "${srcdir}/lib-src/portsmf/allegro.h"; then
+                 PORTSMF_LOCAL_AVAILABLE="yes"
+   else
+                 PORTSMF_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$PORTSMF_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: portSMF library is available in the local tree" >&5
@@ -22675,29 +22317,12 @@
 $as_echo "$as_me: portmidi library is NOT available as system library" >&6;}
    fi
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/portmidi/pm_common/portmidi.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/portmidi/pm_common/portmidi.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/portmidi/pm_common/portmidi.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/portmidi/pm_common/portmidi.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  PORTMIDI_LOCAL_AVAILABLE="yes"
-else
-  PORTMIDI_LOCAL_AVAILABLE="no"
-fi
 
+   if test -f "${srcdir}/lib-src/portmidi/pm_common/portmidi.h"; then
+                 PORTMIDI_LOCAL_AVAILABLE="yes"
+   else
+                 PORTMIDI_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$PORTMIDI_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: portmidi library is available in the local tree" >&5
@@ -22795,29 +22420,11 @@
    fi
 
 
-   as_ac_File=`$as_echo "ac_cv_file_${srcdir}/lib-src/lib-widget-extra/NonGuiThread.h" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ${srcdir}/lib-src/lib-widget-extra/NonGuiThread.h" >&5
-$as_echo_n "checking for ${srcdir}/lib-src/lib-widget-extra/NonGuiThread.h... " >&6; }
-if eval \${$as_ac_File+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "${srcdir}/lib-src/lib-widget-extra/NonGuiThread.h"; then
-  eval "$as_ac_File=yes"
-else
-  eval "$as_ac_File=no"
-fi
-fi
-eval ac_res=\$$as_ac_File
-	       { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
-$as_echo "$ac_res" >&6; }
-if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
-  WIDGETEXTRA_LOCAL_AVAILABLE="yes"
-else
-  WIDGETEXTRA_LOCAL_AVAILABLE="no"
-fi
-
+   if test -f "${srcdir}/lib-src/lib-widget-extra/NonGuiThread.h"; then
+                 WIDGETEXTRA_LOCAL_AVAILABLE="yes"
+   else
+                 WIDGETEXTRA_LOCAL_AVAILABLE="no"
+   fi
 
    if test "$WIDGETEXTRA_LOCAL_AVAILABLE" = "yes"; then
       { $as_echo "$as_me:${as_lineno-$LINENO}: libwidgetextra library is available in the local tree" >&5
